2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    report=[];
+    cities.forEach(element => {
+        cityTemperature = (`The temperature in ${element} is ${temperatureService(element)} degrees`);
+        report.push(cityTemperature);
+    });
+    return report;
 }
 
```

Comment:
for more explanation ( Review )
another solution for more exercises

function getTemperatureReport(cities) {
    let report = [];
    for(let city of cities) {
        let temperature = temperatureService(city);
        report.push(`The temperature in ${city} is ${temperature} degrees`);
    }
    return report;
}